core/domain/fs_services.py

```diff
@@ -21,6 +21,13 @@
 
 
 def get_exploration_file_system_class():
+    """Returns ExplorationFileSystem class to the client if the development mode
+    of constants is on. Otherwise, returns GcsFileSystem.
+
+    Returns:
+        ExplorationFileSystem|GcsFileSystem. Returns the correct file system
```

Comment:
Returns the ... client --> The correct file system class.

Reply:
Done